source/_docs/blueprint/selectors.markdown

```diff
@@ -755,6 +755,27 @@ options:
     List of options that the user can choose from. Small lists (5 items or less), are displayed as radio buttons. When more items are added, a dropdown list is used.
   type: list
   required: true
+multiple:
+  description: >
+    Allows selecting multiple options. If set to `true`, the resulting value of
+    this selector will be a list instead of a single string value.
+  type: boolean
+  required: false
+  default: false
+custom_value:
+  description: >
+    Allows the user to enter and select a custom value (or multiple custom values
+    in addition to the listed options if `multiple` is set to true).
+  type: boolean
+  required: false
+  default: false
+mode:
+  description: >
+    This can be either `list` or `dropdown` mode. When not specified, the frontend
+    decides what input type to use. If `custom_value` is true, this setting will be
+    ignored and the frontend will use a `dropdown` input.
```

Comment:
We write like this above:
> Small lists (5 items or less), are displayed as radio buttons. When more items are added, a dropdown list is used.
We need to clarify `list` means radio buttons, either here or above